backend/src/models/Image.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  url: { primary: true, type: 'string', uri: { allowRelative: true } },
+  alt: { type: 'string' },
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T15:49:12Z" title="Tuesday, February 18th 2020, 4:49:12 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,7 @@
+export default {
+  url: { primary: true, type: 'string', uri: { allowRelative: true } },
+  alt: { type: 'string' },
```


at the moment there is no possibility to add an `alt` attribute and I don't think we'll have it in the near future – do we still want to have it in the `Image` model?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:47:44Z" title="Thursday, February 20th 2020, 8:47:44 pm +01:00">Feb 20, 2020</time>_

---


that is unfortunate, why's that? doesn't that mean our site takes a huge hit for accessibility?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T12:44:02Z" title="Saturday, February 22nd 2020, 1:44:02 pm +01:00">Feb 22, 2020</time>_

---


I considered to remove it from the PR but changed my mind. I have a lot of backend tests where I check whether I can update metadata while re-using the existing image node. The `alt` attribute comes in handy and has quite an obvious benefit for later. It's one of the rare occasions where I'm willing add code anticipating future requirements. Usually I avoid code which does not have an immediate use case.